guides/plugins/plugins/bundle.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+  title: Bundle
+  position: 20
+
+---
+
+# Bundle
+
+Plugins are based on the Symfony bundle concept, but offer additional features like lifecycle events and the ability to be managed in the Shopware administration. This is maybe unwanted in some cases, like project critical customizations which should not be managed via the Shopware administration. In this case, you can use a Symfony bundle instead of a plugin.
```

Comment:
```suggestion
Plugins are based on the Symfony bundle concept, but offer additional features like lifecycle events and the ability to be managed in the Shopware administration.
This is maybe unwanted in some cases, like project critical customizations which should not be managed via the Shopware administration.
In this case, you can use a Symfony bundle instead of a plugin.
```

I find this better readable and also if something needs to be changed, it could be better reviewed, if every sentence is on its own line